### ISSUE TITLE
Object doesn't support `getBoundingClientRect` (found in IE10)

### ISSUE THREAD

**Issue by joeribakker** [dollarshaveclub/stickybits#86]
### Steps to reproduce
1. Apply stickybits to an existing element without providing a `scrollEl` option.
2. Open on IE10 in Windows 7.

### What should happen
Sticky bits galore!

### What actually happens
In IE10 on Windows 7, I'm getting the following error:

```
Object doesn't support property or method `getBoundingClientRect`
```

<img width="1551" alt="screen shot 2017-08-03 at 10 20 32" src="https://user-images.githubusercontent.com/1791066/28912746-cf291614-7835-11e7-8eda-faa60dd5eb12.png">

From looking at the minification it becomes clear that this error is coming from [stickybits.js, line 74](https://github.com/dollarshaveclub/stickybits/blob/master/src/stickybits.js#L70). Since I have initialised without option `scrollEl`, `se` should be `window`.

I don't see any documentation saying that `window` should support `getBoundingClientRect()`, so perhaps `se` should default to `document.documentElement`?

**Comment by yowainwright** [comment]
@joeribakker thanks for noting this bug. I will fix it ASAP. It has been a challenge to support non-`window` stickybits.

**Comment by yowainwright** [comment]
@joeribakker Thanks so much for pointing out this issue! 🙏

**Comment by joeribakker** [comment]
Thanks for the quick response @yowainwright! I solved it in the meantime by passing `window.document.documentElement` in `scrollEl`, so I'll try and remove it after updating to the new version.

**Comment by yowainwright** [comment]
@joeribakker please, keep me posted. 

From history, it looks like the bug was pushed [here](https://github.com/dollarshaveclub/stickybits/commit/7b6c591052a0c26b7880c2de4567037e9c9e3d46#diff-b9cfc7f2cdf78a7f4b91a753d10865a2), then fixed [here](https://github.com/dollarshaveclub/stickybits/commit/5bc76f54eb888348f95d670a356decef29135205#diff-b9cfc7f2cdf78a7f4b91a753d10865a2) but `npm publish` was not run 😑 —until you thankfully pointed out the issue. IE specific tests are coming ASAP to protect against this.

**Comment by DanielRuf** [comment]
It happens in IE 11 (again).

**Comment by DanielRuf** [comment]
```
function stickyTabs(){
  if($('.tabs-component-tabs').length){
    if($('.template-default').length){
      stickybits('.tabs-component-tabs', {useStickyClasses: true, scrollEl: '.template-default'}); // @todo: fix stickybits
    } else if($('.template-content-blue').length) {
      stickybits('.tabs-component-tabs', {useStickyClasses: true, scrollEl: '.template-content-blue'});
    }
  }
}
```

**Comment by DanielRuf** [comment]
`document.querySelector('.template-default').getBoundingClientRect()` works and gives me the desired information.

**Comment by DanielRuf** [comment]
Should we reopen this one here or create a new issue?

**Comment by DanielRuf** [comment]
I am not quite sure. A codepen or unit test for this might be useful.
Can anyone test it and check if https://github.com/dollarshaveclub/stickybits/pull/229/files#diff-eeeaf9f8acfd30c8c8b58939e77b1189R189 fixes this?

**Comment by yowainwright** [comment]
My acceptance tests don't catch issues like this b/c of the setup. I should probably think through this issue deeper ...soon.

**Comment by yowainwright** [comment]
@DanielRuf I think this is fixed as of version 3.0.3. ~Thanks! 🙏
